oemoflex/postprocessing.py

```diff
@@ -45,50 +88,144 @@ def get_capacities(es):
     capacities : pd.DataFrame
         DataFrame containing the capacities.
     """
+
+    def get_facade_attr(attr):
+        # Function constructor for getting a specific property from
+        # the Facade object in bus_results() DataFrame columns "from" or "to"
+        def fnc(flow):
+            # Get property from the Storage object in "from" for the discharge device
+            if isinstance(flow['from'], (facades.TYPEMAP["storage"],
```

Comment:
Same for all occurences.
```suggestion
            if isinstance(flow['from'], (TYPEMAP["storage"],
```